tests/snapshots/install_invalid_tool_configuration.snap

```diff
@@ -4,7 +4,12 @@ assertion_line: 101
 expression: content
 
 ---
-unable to parse Foreman configuration file (at {{CWD}}foreman.toml): data did not match any variant of untagged enum ToolSpec for key `tools.tool` at line 2 column 1
+unable to parse Foreman configuration file (at {{CWD}}foreman.toml): data is not properly formatted for tool:
+
+[tools.tool]
+invalid = "roblox/tooling"
+version = "0.0.0"
+
```

Comment:
I had to change the snapshot. We lose line information since we're longer relying on Serde to deserialize the toml. I figured replacing the incorrect line with the incorrect tool is good enough

Reply:
Yeah, that's probably reasonable, especially given all the cases in which Serde's line annotation falls apart